examples/python-examples/pre_transform_logical_plan_vendor.py

```
@@ -0,0 +1,196 @@
+            pa.field("Running Time min", pa.int32()),
+            pa.field("Distributor", pa.string()),
+            pa.field("Source", pa.string()),
+            pa.field("Major Genre", pa.string()),
+            pa.field("Creative Type", pa.string()),
+            pa.field("Director", pa.string()),
+            pa.field("Rotten Tomatoes Rating", pa.int8()),
+            pa.field("IMDB Rating", pa.float32()),
+            pa.field("IMDB Votes", pa.int64()),
+        ]
+    )
+    
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)